obsRecorder.js

```diff
@@ -177,6 +175,15 @@ function setupScene() {
     };
   }
 
+  // Set all the output source
+  setupSources(scene)
+
+  // Properly scale the video scene using the resolution off whatever monitor is used
+  // The source width is only available after it is set as an output source
+  const realDisplayWidth = osn.Global.getOutputSource(1).source.width;
```

Comment:
I'm getting `undefined` from `osn.Global.getOutputSource(1)` here and everything breaks.

We can access `videoSource` variable here (which should contain plain desktop capture), but `videoSource.width` returns 0 for me here. Weird.